content/rs/administering/database-operations/replica-ha.md

```diff
@@ -0,0 +1,138 @@
+---
+Title: High availability for replica shards
+linkTitle: High availability for replicas
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases: /rs/administering/database-operations/slave-ha.md,
+         /rs/administering/database-operations/slave-ha/,
+         /rs/administering/database-operations/replica-ha.md,
```

Comment:
@lanceleonard Why is the page itself mentioned in aliases here?

Reply:
@banker - It's part of a test I'm doing with redirects.  (We've been seeing some strange behavior and we're trying to isolate the root cause.)